site/src/api/api.ts

```diff
@@ -1603,14 +1603,20 @@ class ApiMethods {
 		return response.data;
 	};
 
+	getGroups = async (): Promise<TypesGen.GroupWithOrganizationInfo[]> => {
+		const response = await this.axios.get("/api/v2/groups");
+		return response.data;
+	};
+
 	/**
 	 * @param organization Can be the organization's ID or name
 	 */
-	getGroups = async (organization: string): Promise<TypesGen.Group[]> => {
+	getGroupsByOrganization = async (
+		organization: string,
+	): Promise<TypesGen.Group[]> => {
 		const response = await this.axios.get(
-			`/api/v2/organizations/${organization}/groups`,
+			`/api/v2/organization/${organization}/groups`,
```

Comment:
Is this correct? It is `organizations` on the Go router.

https://github.com/coder/coder/blob/360d1a10b54242194a675e85607227f4e9f16b52/enterprise/coderd/coderd.go#L306-L306

Reply:
I just noticed this too 😅